lib/Sheet.js

```diff
@@ -716,6 +716,8 @@ class Sheet {
      * @param {string} [opts.text] - The comment text to set.
      * @param {string} [opts.width] - Comment box width.
      * @param {string} [opts.height] - Comment box height.
+     * @param {string} [opts.horizontalAlignment] - Comment horizontalAlignment.
+     * @param {string} [opts.textAlign] - Comment textAlign.
```

Comment:
Very minor, but we should probably keep the same order of the new two options in Cell and Sheet.